README.md

```diff
@@ -91,6 +91,20 @@ Set `lazy: true` in your module options to load Sentry lazily on the client. Thi
 
 You can also pass a lazy config object in your module options (see [options](#lazy) for more information).
 
+## Runtime Configuration
+
+Define a configuration object at runtime in Nuxt config `publicRuntimeConfig[]` to set the environment at runtime for example.
```

Comment:
- It appears that this runtime configuration is only for setting/overriding the options provided in the `config` option. So the one that is passed to `Sentry.init()`. This is not clear at all. The user might think she/he can specify all module options here (which is how it should really work IMO)
- This section could be moved closer to the `Options` section. Probably even be inside of it.
- If making runtime configuration work for "all" options, we should annotate which option is actually possible to override with runtime config since not all options really can. For example, `lazy` can only be resolved during build-time. Same for `disableClientSide`, `disableServerSide` and probably some other.

Reply:
I agree that it should be better specified in documentation.

I didn't want to allow anything else than in the `config` option to be set at runtime because at compile-time this module is doing some work to declare a release, upload the source map, etc. So only a limited set of options can be changed at runtime without breaking the consistency of the configuration, so I think keeping it limited to the `config` option make it clearer.

Reply:
I'm fine if your PR only adds support for overriding the `config` but we can't do it the way you did here as it's not future proof in case we would want to extend it later to support other options.

Also, that solution doesn't support overriding `config` through `clientConfig` or `serverConfig`.

So please change it so that user is required to use this structure:

```js
publicRuntimeConfig: {
  sentry: {
    config: {
      // ...
    }
  }
}
```

You can just support `config`, `clientConfig`, and `serverConfig` and I can later do a follow-up PR to try to support other options.

Reply:
Ok I understand, I'll fix it.